addons/website_sale/models/website_snippet_filter.py

```diff
@@ -93,14 +93,20 @@ def _get_products(self, mode, **kwargs):
         website = self.env['website'].get_current_website()
         search_domain = self.env.context.get('search_domain')
         limit = self.env.context.get('limit')
+        hide_variants = False
+        if search_domain and 'hide_variants' in search_domain:
+            hide_variants = True
```

Comment:
`hide_variants` could be `False` in `search_domain`
```suggestion
            hide_variants = search_domain['hide_variants']
```

Reply:
Done the changes :+1:

Reply:
I just tested and this doesn't work. My bad.

Search domain is in fact a list and `hide_variants` is a key that is only present if its value it `True`. You can revert my comment back to the way it was.

Reply:
I had reverted the changes